ci/check_es.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import argparse
+import elasticsearch
+import sys
+
+def _check_index(server,port,uuid,index):
+    _es_connection_string = str(server) + ':' + str(port)
+    es = elasticsearch.Elasticsearch([_es_connection_string],send_get_body_as='POST')
+    results = es.search(index=index, body={'query': {'match': {'uuid': {'query': uuid, 'operator': 'AND'}}}}, size=1)
```

Comment:
There are 2 fields for uuid, uuid itself which is of type text so analyzers are used and then the keyword subfield which is of type keyword which we can use a term query on. The mapping for uuid looks as follows:
```
        "uuid" : {
          "type" : "text",
          "fields" : {
            "keyword" : {
              "type" : "keyword",
              "ignore_above" : 256
            }
          }
        }
```
So the reason why you're forced to use an AND is because we're using a match query on the text field, instead if we use the [term query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-term-query.html) on the keyword field it'll look return documents where uuid is set to the value we gave. 

So the query should look like: 
```
{
    "query": {
        "term": {
            "uuid.keyword": uuid
        }
    }
}
```
HTH, the reason why we don't want to do the match query is because it'll search for the input uuid in uuid of documents so if the uuid is 1234, then it'll also return documents with uuid 12345. but in reality we only want documents with uuid set to 1234. This is just an edge case but hope it gives an idea of why we should use the term query in this case ( also it's faster because it creates the lookups ).